docs/source/index.rst

```diff
@@ -131,6 +131,7 @@ Topics in this section will help you get started with ExecuTorch.
    export-to-executorch-api-reference
    executorch-runtime-api-reference
    api-life-cycle
+   runtime-python-api-refernece
```

Comment:
Suggest reordering to keep all of the API refs first, then the lifecycle doc last.

Also, the file name was misspelled ("referNEce"), which might be why the docs didn't update for this latest version.

```suggestion
   export-to-executorch-api-reference
   executorch-runtime-api-reference
   runtime-python-api-reference
   api-life-cycle
```